### PR TITLE
docs: add upgrade to Chaos Mesh 2.0

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
Signed-off-by: Liuxiaozhen12 <82579298+Liuxiaozhen12@users.noreply.github.com>
This PR is translated from https://github.com/chaos-mesh/website-zh/pull/62 and https://github.com/chaos-mesh/website-zh/pull/37.

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
It seems the dead link is not in the file that I created. I'm not sure if I need to fix it. PTAL, thanks! @cwen0

File: docs/upgrade-to-2.0.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+tar xvf ./schedule-migration.tar.gz
+```
+
+The `schedule-migration` tool in this package applies only to the Linux x86_64 platform. If you are using other operating systems or architectures, you need to compile the code by yourselves.
+
+## Step 1: Export and upgrade an experiment
+
+You can use the upgrade tool `migrate.sh` to automatically export and upgrade the experiment. Before running, make sure you have enough permissions to access the cluster.
+
+If `migrate.sh` is in the current directory, place the `schedule-migration` tool in this directory and rename it to "main". Then execute the following command to export and upgrade the experiment:
```
Comment (cwen0) [inline docs/upgrade-to-2.0.md:29]: remove `and rename it to "main"`

Comment (cwen0) [reply docs/upgrade-to-2.0.md:29]: https://chaos-mesh.org/website-zh/docs/upgrade-to-2.0#%E7%AC%AC-1-%E6%AD%A5%EF%BC%9A%E5%AF%BC%E5%87%BA%E5%B9%B6%E5%8D%87%E7%BA%A7%E5%AE%9E%E9%AA%8C

Comment (Liuxiaozhen12) [reply docs/upgrade-to-2.0.md]: fixed in https://github.com/chaos-mesh/website/pull/113/commits/886add06dbe1c75424f5dbc5c495dd0773d2c8bd.

File: docs/upgrade-to-2.0.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+In addition, you can use the `schedule-migration` tool to upgrade the specified old version of YAML files:
+
+```bash
+./schedule-migration <path-to-old-yaml> <path-to-new-yaml>
+```
+
+In the YAML file path you have specified, you can get upgraded YAML files. After deleting the old resources, reapply the new YAML files to complete the update process.
+
+## Step 2：Upgrade CRD
```
[the remark this answers is not in the document]
Comment (Liuxiaozhen12) [reply docs/upgrade-to-2.0.md]: fixed in https://github.com/chaos-mesh/website/pull/113/commits/886add06dbe1c75424f5dbc5c495dd0773d2c8bd.

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**